securedrop/static/js/journalist.js

```diff
@@ -61,32 +61,6 @@ $(function () {
       }
     });
 
-  $("#delete-collection").submit(function () {
-    return confirm(get_string("collection-delete-confirm-string"));
-  });
-
-  $("#delete-collections").click(function () {
-    var checked = $(".panel ul#cols li :checkbox").filter(":visible").filter(function(index) {
-        return $(this).prop('checked');
-    });
-    if (checked.length > 0) {
-      return confirm(get_string("collection-multi-delete-confirm-string").supplant({ size: checked.length }));
```

Comment:
All **get_string** must also be revomved from the `securedrop/journalist_templates/js-strings.html` file

Reply:
Makes sense.

Reply:
Resolved this.